docs/pages/material-ui/api/slider.json

```diff
@@ -60,7 +60,7 @@
     "slotProps": {
       "type": {
         "name": "shape",
-        "description": "{ input?: func<br>&#124;&nbsp;object, mark?: func<br>&#124;&nbsp;object, markLabel?: func<br>&#124;&nbsp;object, rail?: func<br>&#124;&nbsp;object, root?: func<br>&#124;&nbsp;object, thumb?: func<br>&#124;&nbsp;object, track?: func<br>&#124;&nbsp;object, valueLabel?: func<br>&#124;&nbsp;{ children?: element, className?: string, open?: bool, style?: object, value?: number, valueLabelDisplay?: 'auto'<br>&#124;&nbsp;'off'<br>&#124;&nbsp;'on' } }"
+        "description": "{ input?: func<br>&#124;&nbsp;object, mark?: func<br>&#124;&nbsp;object, markLabel?: func<br>&#124;&nbsp;object, rail?: func<br>&#124;&nbsp;object, root?: func<br>&#124;&nbsp;object, thumb?: func<br>&#124;&nbsp;object, track?: func<br>&#124;&nbsp;object, valueLabel?: any<br>&#124;&nbsp;func }"
```

Comment:
There shouldn't be change here, the Material UI API's stays intact.

Reply:
It is because of this change in the slotProps type in `SliderUnstyled.d.ts`:
```diff
--- a/packages/mui-base/src/SliderUnstyled/SliderUnstyled.types.ts
+++ b/packages/mui-base/src/SliderUnstyled/SliderUnstyled.types.ts
@@ -171,7 +171,7 @@ export interface SliderUnstyledOwnProps {
       SliderUnstyledOwnerState
     >;
     valueLabel?: SlotComponentProps<
-      typeof SliderValueLabelUnstyled,
+      React.ElementType,
       SliderUnstyledComponentsPropsOverrides,
       SliderUnstyledOwnerState
     >;
```
`SliderValueLabelUnstyled` component is removed and the Material UI Slider types extend the SliderUnstyled types. 
Is there any way? There is one `DeepOmit` utility: https://github.com/ts-essentials/ts-essentials#deepomit.